downstream/modules/hub/proc-create-content-developers.adoc

```diff
@@ -2,24 +2,24 @@
 
 = Creating a new group for content curators
 
-You can create a new group in {HubName} designed to support content curation in your organization that contributes internally developed collections for publication in {HubName}.
+You can create a new group in {HubName} designed to support content curation in your organization. This group can contribute internally developed collections for publication in {HubName}.
 
-Use this procedure to create and edit a group and assign the required permissions to help content developers create namespaces and upload their internally developed collections to {HubName}.
+To help content developers create namespace and upload their internally developed collections to {HubName}, you must first create and edit a group and assign the required permissions.
 
 .Prerequisites
 
-* You have administrative permissions in {HubName} and create groups.
+* You have administrative permissions in {HubName} and can create groups.
 
 .Procedure
 
 . Log in to your local {HubName}.
-. From the navigation panel, select menu:User Access[Groups] and click btn:[Create].
-. Enter *Content Engineering* as a *Name* for the group in the modal and click btn:[Create]. You have created the new group and the *Groups* page appears.
+. Navigate to menu:User Access[Groups] and click btn:[Create].
```

Comment:
We decided that the correct form for this is: "From the navigation panel, select menu:item1[]..."